dvc/repo/collect.py

```diff
@@ -49,7 +49,6 @@ def _collect_paths(
                 )
             else:
                 logger.warning("'%s' was not found at: '%s'.", path_info, rev)
-            continue
```

Comment:
Removing - even if we want to have a warning that some file does not exist, we need to try to open that file later, so that we get `FileNotFound` in results, for API consistiency.